lyte/lyte.js

```diff
@@ -22,6 +22,7 @@ ly.te = function() {
 			if (thumb !== "") {
 				bgCss="#"+bgId+" { background-image: url("+thumb+"); }";
 				ly.addCss(bgCss);
+				ly.addCss("#"+bgId+".wyl-lazy { background-image: none; background-color: #ccc; }");
```

Comment:
but isn't the background-image loaded due to the CSS from line 23?

Reply:
no, because the wyl-lazy class is applied by default in the patches for wp-youtube-lyte.php when the page is loaded, so the CSS is not used until wyl-lazy class gets removed by our lazyloader script